pkg/run/run.go

```diff
@@ -11,6 +11,9 @@ import (
 	utilglog "github.com/openshift/source-to-image/pkg/util/glog"
 )
 
+// DefaultEntrypoint is the default entry point used when starting containers
+const DefaultEntrypoint = "/bin/env"
```

Comment:
`var DefaultEntrypoint = []string{"/bin/env"}` ?

(you don't have to create the arrays when calling this)